### ISSUE TITLE
[react-native-payments][android] ADDITION OF GOOGLE PAY AS A PAYMENT METHOD

### ISSUE THREAD

**Issue by matheusjs-cit** [rnw-community/rnw-community#256]
support for Google Pay as a payment method option within the library. With these additions, developers can now easily integrate Google Pay into their applications, providing a smoother and more modern payment experience for users.
Interface AndroidPaymentMethodDataDataInterface
Interface AndroidPaymentDataRequest
Class PaymentRequest
This implementation not only adds a new payment method but also enhances the library's flexibility, allowing developers to offer a more comprehensive payment experience.

I look forward to your feedback and suggestions for improvements. Thank you for considering this addition!

**Comment by vitalyiegorov** [comment]
@matheusjs-cit But this library already does it for the android, or am i missing something?

**Comment by vitalyiegorov** [comment]
@matheusjs-cit Absolutely, as long as we comply with all platforms we can add it, please provide a PR and I will do review.

**Comment by vitalyiegorov** [comment]
Fork this repo and then create a pull request @matheusjs-cit  https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

**Comment by vitalyiegorov** [comment]
@matheusjs-cit Please open a fork PR when you are ready
